app/lib/features/account_deletion_config.rb

```diff
@@ -14,31 +15,18 @@ def self.config
       configuration.config
     end
 
-    def self.valid?
-      configuration.valid?
+    def self.enabled?
+      configuration.enabled?
     end
 
     def initialize(config = {})
-      format_config(config)
-      check_validity(config)
+      @config = ActiveSupport::OrderedOptions.new.merge((config.presence || {}).symbolize_keys)
     end
 
-    attr_reader :config, :valid
-    alias valid? valid
-
-    private
-
-    def format_config(config)
-      config = (config || {}).each_with_object({}) do |(key, value), collection|
-        collection[key.to_sym] = value if value.is_a?(Integer)
-      end
-      @config = ActiveSupport::OrderedOptions.new.merge(config)
-    end
+    attr_reader :config
 
-    def check_validity(initial_config)
-      @valid = %i[account_suspension account_inactivity contract_unpaid_time].all? { |key| config.key?(key) }
-      return if valid || initial_config.blank?
-      Rails.logger.warn '[WARNING] Can\'t enable "automatic inactive tenant account deletion". Please revise your config"'
+    def enabled?
+      config.any?
```

Comment:
Hmm I see. 
Well in all our other features, the file always has an option `enabled` https://github.com/3scale/porta/blob/master/config/examples/redhat_customer_portal.yml#L2

I think we should follow that

So this is just delegating to `config.enabled`